force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -290,7 +310,9 @@ public with sharing class VariablesDataTypesOperators {
      * @return A Decimal representing the monthly paycheck.
      */
     public static Decimal calculateMonthlyPaycheck(Decimal hourlyRate, Double numberOfHours) {
-        return null; // Replace null with the variable you used to store the result
+        Decimal weeklyPay = hourlyRate * numberOfHours;
+        Decimal monthlyPay = weeklyPay * 4;
```

Comment:
Try creating a variable for the number 4 so it is not a magic number.

Reply:
Since the number of weeks are constant, hard coded the value. Will create a variable.